vm/src/builtins/pytype.rs

```diff
@@ -239,6 +239,19 @@ impl PyTypeRef {
 
 #[pyimpl(with(SlotGetattro, SlotSetattro, Callable), flags(BASETYPE))]
 impl PyType {
+    // bound method for every type
+    pub(crate) fn __new__(zelf: PyRef<Self>, args: FuncArgs, vm: &VirtualMachine) -> PyResult {
```

Comment:
`__new__` is sort of confusing to me, I'd prefer if this kept `tp_new_wrapper` so we can easily see which function this maps to in CPython.

Reply:
This is a body of bound method `__new__` just like `mro`. The only difference of them is `__new__` is a bound method for every type including `type` but `mro` is an unbound method for `type` and bound as boundmethod for other types.
I think we'd better to keep this name as same as other methods when it works like other methods.